iceoryx_hoofs/include/iceoryx_hoofs/cxx/optional.hpp

```diff
@@ -185,6 +170,7 @@ class optional : public FunctionalInterface<optional<T>, T, void>
 
     /// @brief Will return true if the optional contains a value, otherwise false.
     /// @return true if optional contains a value, otherwise false
+    // AXIVION Next Construct AutosarC++19_03-A13.5.3: requires a user-defined conversion
```

Comment:
This is in my opinion not a valid justification. Please remember to write (when it is fitting) why we are doing it and why it is safe.
One example would be:
Implemented to be as close to the STL interface as possible. In combination with the keyword explicit accidental casts can be excluded and the usage is well known in the C++ community.